### ISSUE TITLE
Cost diff discrepancy on Premium Azure Event Hub service

### ISSUE THREAD

**Issue by tim775** [infracost/infracost#1354]
### Cost diff discrepancy on Azure Event Hub service
Azure Event Hub change Standard to Premium SKU and Infracost shows decrease in price, pretty sure the increase is actually quite big. Result of diff:
```
Detected Terragrunt directory at workloads/development/australiaeast/eventhub
  ✔ Running terragrunt run-all terragrunt-info 
  ✔ Running terragrunt run-all plan 
  ✔ Running terragrunt show 
  ✔ Extracting only cost-related params from terragrunt plan 
  ✔ Retrieving cloud prices to calculate costs 

Project: gysiedebruyn/terraform-azure/workloads/development/australiaeast/eventhub

~ azurerm_eventhub_namespace.ns
  -$21.90 ($21.90 → $0.00)

    - Ingress event (Standard)
      Monthly cost depends on usage
        -$0.028 per 1M events

    ~ Throughput
      -$21.90 ($21.90 → $0.00)

Monthly cost change for gysiedebruyn/terraform-azure/workloads/development/australiaeast/eventhub
Amount:  -$21.90 ($29.70 → $7.80)
Percent: -74%

──────────────────────────────────
Key: ~ changed, + added, - removed

4 cloud resources were detected, rerun with --show-skipped to see details:
∙ 3 were estimated, 2 include usage-based costs, see https://infracost.io/usage-file
∙ 1 was free
```

This is the actual Terraform plan output:
```
  # azurerm_eventhub_namespace.ns must be replaced
-/+ resource "azurerm_eventhub_namespace" "ns" {
      ~ default_primary_connection_string         = (sensitive value)
      + default_primary_connection_string_alias   = (sensitive value)
      ~ default_primary_key                       = (sensitive value)
      ~ default_secondary_connection_string       = (sensitive value)
      + default_secondary_connection_string_alias = (sensitive value)
      ~ default_secondary_key                     = (sensitive value)
      ~ id                                        = "/.../jdb-tf-standard-hub" -> (known after apply)
      ~ maximum_throughput_units                  = 0 -> (known after apply)
        name                                      = "jdb-tf-standard-hub"
      ~ network_rulesets                          = [
          ~ {
              ~ virtual_network_rule           = [] -> null
                # (3 unchanged elements hidden)
            },
        ]
      ~ sku                                       = "Standard" -> "Premium" # forces replacement
        tags                                      = {
            "CostCentre" = "PoC"
            "Module"     = "eventhub"
            "Owner"      = "Jan de Bruyn"
        }
      ~ zone_redundant                            = false -> true # forces replacement
        # (4 unchanged attributes hidden)
    }
```

_Originally posted by @gysiedebruyn in https://github.com/infracost/infracost/discussions/1271#discussioncomment-2137601_

**Comment by tim775** [comment]
This will be released in v0.9.18

**Comment by vdmgolub** [comment]
This is live in v0.9.18.
